### PR TITLE
Added Purpur Eggs and Updated README.MD

### PR DESCRIPTION
**Pull request by RehanPlayz**
(I haven't tested this)

### CODE REVIEW

**Comment by lrmtheboss** [comment]
hay, could you update your eggs to use the new yolks images, please?

ghcr.io/pterodactyl/yolks:java_17
ghcr.io/pterodactyl/yolks:java_16
ghcr.io/pterodactyl/yolks:java_11
ghcr.io/pterodactyl/yolks:java_8

**Comment by RehanPlayz** [comment]
> hay, could you update your eggs to use the new yolks images, please?
> 
> ghcr.io/pterodactyl/yolks:java_17 ghcr.io/pterodactyl/yolks:java_16 ghcr.io/pterodactyl/yolks:java_11 ghcr.io/pterodactyl/yolks:java_8

sure

**Comment by RehanPlayz** [comment]
@lrmtheboss Could you test this and merge it please

**Comment by RehanPlayz** [comment]
> @lrmtheboss Could you test this and merge it please
NVM you arent a maintainer

**Comment by RehanPlayz** [comment]
bump

**Comment by Grizzly90** [comment]
How about testing the egg yourself? You can't expect anyone do the work, except if someone requested the egg. In that case you can give the egg to this person to test it.

**Comment by RehanPlayz** [comment]
> How about testing the egg yourself? You can't expect anyone do the work, except if someone requested the egg. In that case you can give the egg to this person to test it.

Hey, I don't have a Ptero environment to test the egg on, It should in theory work without any issues and nobody really requested it, I just wanted to make a contribution

**Comment by RehanPlayz** [comment]
> > How about testing the egg yourself? You can't expect anyone do the work, except if someone requested the egg. In that case you can give the egg to this person to test it.
> 
> Hey, I don't have a Ptero environment to test the egg on, It should in theory work without any issues and nobody really requested it, I just wanted to make a contribution

I basically just edited the Geyser + Paper egg and just replaced some stuff

File: egg-purpur--geyser--floodgate.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    "author": "support@geysermc.org",
+    "description": "High performance Spigot fork that aims to fix gameplay and mechanics inconsistencies.\r\n\r\nThis customized egg will install the GeyserMC and Floodgate plugins as well. This allows Bedrock players to connect to the server without needing a Java account.",
+    "features": [
+        "eula"
+    ],
+    "images": [
+        "ghcr.io/pterodactyl/yolks:java_17",
+        "ghcr.io/pterodactyl/yolks:java_16",
+        "ghcr.io/pterodactyl/yolks:java_11",
+        "ghcr.io/pterodactyl/yolks:java_8",
```
Comment (7man7LMYT) [inline egg-purpur--geyser--floodgate.json:18]: There's a comma at the end of the java_8 part that shouldn't be there which results in this failing to import into Pterodactyl.

Comment (RehanPlayz) [reply egg-purpur--geyser--floodgate.json:18]: Oh, I didn't notice that

**Review by 7man7LMYT — CHANGES_REQUESTED** [review]

**Comment by onebeastchris** [comment]
Since this PR has had no activity for 1.5 years now, I'm closing it - feel free to address the review and re-open it :)

**Pull request closed without merging**